### PR TITLE
Feature optimize exceptions tests

### PR DESCRIPTION
**Pull request by caroscrx**

### CODE REVIEW

File: src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoUiRuntimeExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        assertThat(new TodoUiRuntimeException()).hasMessage(null).hasCause(null);
+    }
+
+    @Test
+    void runTimeException() {
+        assertThat(new TodoUiRuntimeException("sjs")).hasMessage("sjs").hasCause(null);
+    }
+
+    @Test
+    void runTimeExceptionWithCause() {
```
Comment (termitehuegel) [inline src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoUiRuntimeExceptionTest.java:19]: rename to runTImeExceptionWithMessage

File: src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoUiRuntimeExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        assertThat(new TodoUiRuntimeException("test", null)).hasMessage("test").hasCause(null);
+    }
+
+    @Test
+    void runTimeExceptionNothingCause() {
```
Comment (termitehuegel) [inline src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoUiRuntimeExceptionTest.java:24]: runTimeExceptionWithCause

File: src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoUiRuntimeExceptionTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+        assertThat(new TodoUiRuntimeException(new RuntimeException())).hasCause(new RuntimeException());
+    }
+
+    @Test
+    void runTimeExceptionAll() {
+
+        assertThat(new TodoUiRuntimeException("test", null, true, true)).hasMessage("test").hasCause(null);
```
Comment (termitehuegel) [inline src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoUiRuntimeExceptionTest.java:32]: remove new line

File: src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoRuntimeExceptionTest.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    @Test
+    void runTimeException(){
+
+        assertThat(new TodoRuntimeException("sjs")).hasMessage("sjs").hasCause(null);
+    }
+
+    @Test
+    void runTimeExceptionWithCause(){
```
Comment (termitehuegel) [inline src/test/java/hwr/oop/group4/todo/commons/exceptions/TodoRuntimeExceptionTest.java:22]: message

**Review by termitehuegel — CHANGES_REQUESTED** [review]

**Review by yxyx-github — APPROVED** [review]

**Review by yxyx-github — APPROVED** [review]

**Review by termitehuegel — APPROVED** [review]

**Pull request merged**